pkg/flowaggregator/clickhouseclient/clickhouseclient.go

```diff
@@ -0,0 +1,511 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clickhouseclient
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/ClickHouse/clickhouse-go"
+	"github.com/gammazero/deque"
+	ipfixentities "github.com/vmware/go-ipfix/pkg/entities"
+	"k8s.io/klog/v2"
+)
+
+const (
+	maxQueueSize   = 1 << 19 // 524288. ~500MB assuming 1KiB per record
+	commitInterval = 1 * time.Second
+	insertQuery    = `INSERT INTO flows (
```

Comment:
I think we may need to assign value to the column `timeInsterted` in the flow aggregator during the insertion, as the default value assigned by Clickhouse would be `1970-01-01 00:00:00`, which leads to the deletion of all the records when we use TTL mechanism.

Reply:
The value assignment has been taken care of by the latest data schema. https://github.com/antrea-io/antrea/blob/af490d89ec1eed6a4b74d252eaa3bab5a91b4ad1/build/yamls/flow-visibility/flow-visibility.yaml#L209

Thanks Yanjun for checking it.